src/current/v23.1/goldengate.md

```diff
@@ -0,0 +1,530 @@
+1. Set up the `ODBC.ini` file for the Oracle GoldenGate host:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    # This is needed so that OGG knows where to look for connection details for the database
+    export ODBCINI=/etc/odbc.ini
+
+    vi /etc/odbc.ini
+    ~~~
+        
+1. Set up the CockroachDB {{ site.data.products.serverless }} parameters:
+    - Replace the login details with your own. Be sure to prefix the database name with `{hostname}`.
+    - Make sure your {{ site.data.products.serverless }} cluster's [root CA certificate](https://cockroachlabs.com/docs/cockroachcloud/connect-to-a-serverless-cluster#connect-to-your-cluster) is in the `TrustStore` path.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    [CRDBSERVERLESS]
+    # The following driver will always point to your Oracle GoldenGate for PostgreSQL installation
+    Driver=/u01/ggs-pg/lib/GGpsql25.so
+    Description=DataDirect 7.1 PostgreSQL Wire Protocol
+    Database={hostname}.{database}
+    HostName={host}
+    PortNumber=26257
+    LogonID={your sql user}
+    Password={your sql user's pass}
+    EncryptionMethod=1 
+    ValidateServerCertificate=1 
+    TrustStore=/root/.postgresql/root.crt
+    ~~~
+        
+1. Log in to the database:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd /u01/ggs-pg
+
+    ./ggsci
+
+    # To log into the local database
+    DBLOGIN SOURCEDB CRDBLOCAL
+
+    # To log into the serverless database and then enter your password
+    DBLOGIN SOURCEDB CRDBSERVERLESS
+    ~~~     
+
+## Set up Extract to capture data from a source database
+
+[Extract](https://docs.oracle.com/goldengate/c1230/gg-winux/GGCON/processes-and-terminology.html) is Oracle GoldenGate's data capture mechanism that is configured to run against the source database.
+
+{{site.data.alerts.callout_success}}
+The steps in this section should be run on a machine and in a directory where Oracle GoldenGate for Oracle is installed.
+{{site.data.alerts.end}}
+
+1. Inside the GGSCI terminal, create and open the `epos` parameter file for the Oracle source:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    cd /u01/ggs # Otherwise known as $OGG_HOME
+    ./ggsci
+
+    edit param epos
+    view param epos
+    ~~~
+
+1. Edit the following parameters:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~
+    # This section will run an extract on the source Oracle DB and then send that data over to the trail file at `./dirdat/ab` on the remote host.
+    EXTRACT epos
+    USERIDALIAS gg_source
+    RMTHOST {host-name}, MGRPORT {port}
+    RMTTRAIL ./dirdat/ab
+    TABLE OGGADM1.testtable;
+    ~~~
+
+1. Back in the GGSCI terminal, run the following and start the Extract service:
+
+    {% include_cached copy-clipboard.html %}
+    ~~~ shell
+    add extract epos, tranlog, begin now
+    add rmttrail ./dirdat/ab, extract epos megabytes 10
+
+    # This is crucial in order to address the logmining not found error
```

Comment:
This kind of info might get lost if it is in a comment. Consider adding it near line 158 instead.